breadbox/breadbox/startup.py

```diff
@@ -96,31 +94,3 @@ async def __call__(self, scope, receive, send):
         await self.app(scope, receive, send)
 
 
-class GCPExceptionReporter:
```

Comment:
Weird, we had a GCPExceptionReporter in startup.py already? I guess this wasn't being used?

Reply:
Sorry for the confusion, this definition was being used before, I just moved it to a different file now.

This `GCPExceptionReporter` class had previously been located in `startup.py`, which I tend to think of as being specific to the breadbox app - so it felt a little more natural to me to put it in with the logging utilities, now that it's being used by the background worker as well. I'm not sure it matters much to me one way or another though, so let me know if you have an opinion.